Tests/MapboxMapsTests/Snapshot/Mocks/MockMapSnapshotter.swift

```diff
@@ -0,0 +1,87 @@
+@testable import MapboxMaps
+import MapboxCoreMaps
+@_implementationOnly import MapboxCoreMaps_Private
+@_implementationOnly import MapboxCommon_Private
+
+final class MockMapSnapshotter: MockStyleManager, MapSnapshotterProtocol {
+
+    @Stubbed var style: Style?
+    @Stubbed var options: MapboxCoreMaps.MapSnapshotOptions?
+    @Stubbed var size: Size?
+    @Stubbed var image: UIImage?
+    @Stubbed var coordinateBoundsForCamera: CoordinateBounds?
+    @Stubbed var cameraForCoordinates: MapboxCoreMaps.CameraOptions?
```

Comment:
I couldn't find anywhere that these are needed, except [line 11](https://github.com/mapbox/mapbox-maps-ios/pull/1536/files#diff-d34a899d1319ae447aa830b9d6d040f8cd37be9dbcb74992ee68199591362f1bR11)